pkg/operator/genericoperatorclient/dynamic_operator_client_test.go

```diff
@@ -29,10 +30,52 @@ func TestSetOperatorSpecFromUnstructured(t *testing.T) {
 			},
 			expected: map[string]interface{}{
 				"spec": map[string]interface{}{
-					"non-standard-field": "value",
-					"logLevel":           "Trace",
-					"managementState":    "",
-					"operatorLogLevel":   "",
+					"non-standard-field":         "value",
+					"logLevel":                   "Trace",
+					"managementState":            "",
+					"operatorLogLevel":           "",
+					"unsupportedConfigOverrides": nil,
+					"observedConfig":             nil,
+				},
+			},
+		},
+		{
+			name: "replace-rawextensions",
+			in: map[string]interface{}{
+				"spec": map[string]interface{}{
```

Comment:
Do we need some tests with both spec and status set?  Along with metadata?

Reply:
Kind of out-of-scope, but added anyway.